app/Http/Controllers/Admin/MeetingsController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Models\Room;
+use App\Models\Level;
+use App\Models\Meeting;
+use App\Models\Category;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class MeetingsController extends Controller
+{
+    const DEFAULT_STATUS_ID = 1;
+    private $meeting;
+    public function __construct(Meeting $meeting)
+    {
+        $this->meeting = $meeting;
+        $this->meeting->updateStatus();
```

Comment:
what is the purpose of this `updateStatus()`?

Reply:
It was defined in MeetingModel.
The function is to update status_id in meetings table.
If meeting start_at time is over, change the status_id 1(stand-by) to 2(done).